spec/features/users/login_spec.rb

```diff
@@ -93,4 +93,45 @@
       expect(page).to have_content("Welcome, #{merchant.name}, you are logged in!")
     end
   end
+  describe "all users can log out" do
+      it 'can log out as a registered user' do
+      user = User.create(name: "Jordan",
+                          address: "394 High St",
+                          city: "Denver",
+                          state: "CO",
+                          zip_code: "80602",
+                          email: "hotones@hotmail.com",
+                          password: "password",
+                          password_confirmation: "password",
+                          role: 0)
+
+      visit '/'
+
+      click_link "Login"
+      expect(current_path).to eq('/login')
+
+      fill_in :email, with: user.email
+      fill_in :password, with: user.password
+
+      click_button "Login"
+
+      expect(current_path).to eq('/profile')
```

Comment:
These expects (current_path through the nav bar expects) aren't necessary, we've already tested these elsewhere!